src/sdk/cpp/examples/callback_mode/callback_mode.cpp

```diff
@@ -38,10 +37,14 @@ void signal_handler(int) {
     client.Stop();
 }
 
+static auto logger = NamedLogger{"CallbackMode"};
+
 int main(int, char**) {
+    logger.Error() << "Fired up";
+
     // Register a global error handler
     client.OnError = [](const ErrorMessage& msg) {
-        logging::Error() << "Something went a awry, " << msg.GetMessage(); 
+        logger.Error() << "Something went a awry, " << msg.GetMessage();
```

Comment:
idea: To have the same look and feel you could provide a "GetLogger" function by the sdk for such use cases but I'm fine with that as well.

Reply:
There is a `GetLogger` method available to Talents, but since we don't have direct access to the underlying talent in callback mode we use the global logger. It is possible to create a `NamedLogger` if one wants to logs to have a particular "tag".

```c++
NamedLogger my_logger{"callback_mode"};
```